### ISSUE TITLE
Contract in `@openzeppelin/contracts-upgradeable` raises an `internal-function-storage` error

### ISSUE THREAD

**Issue by clemsos** [OpenZeppelin/openzeppelin-upgrades#1037]
After upgrading to the latest version of the upgrades plugin, a `internal-function-storage` error is thrown with contract from the `@openzeppelin/contracts-upgradeable` lib. Here is the line:

https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/e49c574f71d402bbd5905175ec2205af02e87dda/contracts/governance/utils/VotesUpgradeable.sol#L259-L262

The error 

```
@openzeppelin/contracts-upgradeable5/governance/utils/VotesUpgradeable.sol:261: Variable `op` is an internal function
    Use external functions or avoid functions in storage.
     If you must use internal functions, skip this check with the `unsafeAllow.internal-function-storage`
     flag and ensure you always reassign internal functions in storage during upgrades
    https://zpl.in/upgrades/error-009
      at assertUpgradeSafe (/home/unlock/node_modules/@openzeppelin/upgrades-core/src/validate/query.ts:20:11)
      at validateImpl (node_modules/@openzeppelin/hardhat-upgrades/src/utils/validate-impl.ts:42:20)
      at validateProxyImpl (node_modules/@openzeppelin/hardhat-upgrades/src/utils/validate-impl.ts:63:10)
      at deployProxyImpl (node_modules/@openzeppelin/hardhat-upgrades/src/utils/deploy-impl.ts:72:3)
      at Proxy.deployProxy (node_modules/@openzeppelin/hardhat-upgrades/src/deploy-proxy.ts:48:28)
      at Context.<anonymous> (test/votes.js:36:10)
```

What is the best approach to silence the warning ? I would prefer to not disable the check for the entire upgrade if possible.

Many thanks !

**Comment by ericglau** [comment]
This is a false positive, as the check was intended to give the error if there is an internal function pointer in storage variables.  In this case, internal function pointers are used directly from the implementation code and not from storage variables.

We are working on a fix for the validations, but for now you can disable the `internal-function-storage` check when calling the Upgrades plugin's functions according to
https://docs.openzeppelin.com/upgrades-plugins/1.x/faq#how-can-i-disable-checks

**Comment by ericglau** [comment]
This is fixed in [@openzeppelin/upgrades-core@1.34.1](https://github.com/OpenZeppelin/openzeppelin-upgrades/releases/tag/%40openzeppelin%2Fupgrades-core%401.34.1), you can update your dependencies to use that package which is a transitive dependency of the Hardhat Upgrades plugin.

For example: `npm install @openzeppelin/upgrades-core@latest`

**Comment by clemsos** [comment]
Thank you very much for the help and the reactivity :)
